lib/ui/io_manager.h

```diff
@@ -24,7 +24,8 @@ class IOManager {
 
   virtual fml::RefPtr<flutter::SkiaUnrefQueue> GetSkiaUnrefQueue() const = 0;
 
-  virtual std::shared_ptr<fml::SyncSwitch> GetIsGpuDisabledSyncSwitch() = 0;
+  virtual const std::shared_ptr<fml::SyncSwitch>
+  GetIsGpuDisabledSyncSwitch() = 0;
```

Comment:
This should be `std::shared_ptr<const fml::SyncSwitch>` here and everywhere else.